core/src/main/java/hudson/model/TimeZoneProperty.java

```diff
@@ -0,0 +1,82 @@
+package hudson.model;
+
+import hudson.Extension;
+import hudson.util.ListBoxModel;
+import hudson.util.ListBoxModel.Option;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import javax.annotation.CheckForNull;
+import java.io.IOException;
+import java.util.TimeZone;
+
+public class TimeZoneProperty extends UserProperty implements Saveable {
```

Comment:
Would be nice to add some Javadoc to the new class

Reply:
added